Source/FullScreenPokemon.ts

```diff
@@ -2497,6 +2509,7 @@ module FullScreenPokemon {
             if (other.dialogOptions) {
                 this.animateCharacterDialogOptions(thing, other, other.dialogOptions);
             } else if (other.trainer && !(<IEnemy>other).alreadyBattled) {
+                this.autoSave();
```

Comment:
You can probably remove this line right? There's an autoSave call at the end of the function anyway.